Companion app sensors stopped updating when not connected to home Wi-Fi (405 method not allowed)

<!-- READ THIS FIRST:
- Make sure you run the latest version of the Android app
- Make sure you run the latest version of Home Assistant
- Make sure to check the Companion docs for troubleshooting and configuration: https://companion.home-assistant.io/
- Make sure the bug you found is not already reported, we love to put work in bugfixes instead of closing duplicate bug reports
  DO NOT DELETE ANY TEXT from this template! All requested information is important.
-->

<!-- If you are reporting an issue for Wear OS please list both devices for the below 3 fields -->
**Home Assistant Android app version(s):**

**Android version(s):** 2025.12.3-full

**Device model(s):** OnePlus 10 Pro

**Home Assistant version:** 2025.12.3

**Last working Home Assistant release (if known):** 2023.12.3

**Description of problem, include YAML if issue is related to notifications:**
For the past two days, the companion app has been unable to update any of the sensors associated with my phone in home assistant, including the location sensor, whenever I am disconnected from my home Wi-Fi. 

While doing some digging, I discovered that every time the app attempts to update a sensor, it receives a 405 error code along with the error message "405: Method Not Allowed". As far as I can tell, this issue appeared after updating the android app from version 2025.12.2-full to 2025.12.3-full. 

<!--
- For Wear OS devices we will need the LogCat logs from the device.
- For Android Auto the logs can be retrieved from the connected device.
- Logs from the device can be taken from Settings > Companion App Troubleshooting > Show and Share Logs
-->
**Companion App Logs:**

```
--------- beginning of system
12-24 00:15:26.817 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:15:26.843 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:15:26.875 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:15:56.249 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:15:57.896 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:16:42.744 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:16:42.745 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:16:43.572 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:16:59.367 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:16:59.387 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:16:59.445 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:17:01.841 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:17:01.871 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:17:02.687 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:19:05.916 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:19:05.925 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:19:05.985 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:19:17.425 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:19:17.455 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:19:18.234 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:19:20.930 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:19:20.939 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:19:20.973 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:19:22.770 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:19:22.805 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
12-24 00:19:23.609 11512 11512 W VRI[WebViewActivity]: handleResized abandoned!
--------- beginning of main
12-24 00:20:37.911 11512 15693 D OplusScrollToTopManager: io.homeassistant.companion.android/io.homeassistant.companion.android.webview.WebViewActivity,This com.android.internal.policy.DecorView{30e8ef4 V.E...... R....... 0,0-1440,3216 aid=14 alpha=1.0 viewInfo = }[WebViewActivity] change focus to false
12-24 00:20:38.177 11512 11512 D callGcSupression: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object[])' on a null object reference
12-24 00:20:38.177 11512 11512 D ScrollOptim [SceneManager]: updateCurrentActivity: mCurrentActivityName=null, isOptEnable=true, isAnimAheadEnable=true, isFrameInsertEnable=true, InsertNum=1, isEnabledForScrollChanged=false
12-24 00:20:38.190 11512 11512 D IntegrationRepositoryImpl: setAppActive(): false
12-24 00:20:38.208 11512 20530 D VRI[WebViewActivity]: dispatchAppVisibility visible:false
12-24 00:20:38.212 11512 11512 D SensorReceiverBase: Received intent: io.homeassistant.companion.android.UPDATE_SENSORS
12-24 00:20:38.213 11512 11512 D VRI[WebViewActivity]: visibilityChanged oldVisibility=true newVisibility=false
12-24 00:20:38.239 11512 11512 D HWUI    : RenderProxy::destroy: this=0xb400007857e8e200, mContext=0xb4000077c04d63c0
12-24 00:20:38.239 11512 15619 D HWUI    : SkiaOpenGLPipeline::setSurface: this=0xb40000787b52b080, surface=NULL
12-24 00:20:38.241 11512 15619 D BufferQueueProducer: [VRI[WebViewActivity]#72(BLAST Consumer)72](id:2cf800000064,api:1,p:11512,c:11512) disconnect: api 1
12-24 00:20:38.243 11512 11512 D ViewRootImplExtImpl: wrapConfigInfoIntoFlags rotation=0, smallestScreenWidthDp=411, relayoutAsync=false, flags=0, newFlags=26935296, residentWS=false, title=io.homeassistant.companion.android/io.homeassistant.companion.android.webview.WebViewActivity
12-24 00:20:38.256 11512 11512 D VRI[WebViewActivity]: relayoutWindow result, sizeChanged:false, surfaceControlChanged:true, transformHintChanged:false, mSurfaceSize:Point(1440, 3216), mLastSurfaceSize:Point(1440, 3216), mWidth:1440, mHeight:3216, requestedWidth:1440, requestedHeight:3216, transformHint:0, installOrientation:0, displayRotation:0, isSurfaceValid:false, attr.flag:-2122251904, tmpFrames:ClientWindowFrames{frame=[0,0][1440,3216] display=[0,0][1440,3216] parentFrame=[0,0][0,0]}, relayoutAsync:false, mSyncSeqId:0
12-24 00:20:38.257 11512 11512 D BLASTBufferQueue: [VRI[WebViewActivity]#72](f:0,a:1) destructor()
12-24 00:20:38.257 11512 11512 D BufferQueueConsumer: [VRI[WebViewActivity]#72(BLAST Consumer)72](id:2cf800000064,api:0,p:-1,c:11512) disconnect
12-24 00:20:38.264 11512 11512 D callGcSupression: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object[])' on a null object reference
12-24 00:20:38.268 11512 11512 W WindowOnBackDispatcher: sendCancelIfRunning: isInProgress=false callback=androidx.navigationevent.OnBackInvokedInput$createOnBackAnimationCallback$1@10081a3
12-24 00:20:38.268 11512 11512 D VRI[WebViewActivity]: setWindowStopped stopped:true
12-24 00:20:38.271 11512 11512 V AutofillManager: updateSessionLocked(): id=null, bounds=null, value=null, action=5, flags=0
12-24 00:20:38.331 11512 11512 D ViewRootImplExtImpl: wrapConfigInfoIntoFlags rotation=0, smallestScreenWidthDp=411, relayoutAsync=false, flags=0, newFlags=26935296, residentWS=false, title=io.homeassistant.companion.android/io.homeassistant.companion.android.webview.WebViewActivity
12-24 00:20:38.337 11512 11512 D VRI[WebViewActivity]: relayoutWindow result, sizeChanged:false, surfaceControlChanged:true, transformHintChanged:false, mSurfaceSize:Point(1440, 3216), mLastSurfaceSize:Point(1440, 3216), mWidth:1440, mHeight:3216, requestedWidth:1440, requestedHeight:3216, transformHint:0, installOrientation:0, displayRotation:0, isSurfaceValid:false, attr.flag:-2122251904, tmpFrames:ClientWindowFrames{frame=[0,0][1440,3216] display=[0,0][1440,3216] parentFrame=[0,0][0,0]}, relayoutAsync:false, mSyncSeqId:0
12-24 00:20:38.634 11512 22191 D ServerConnectionStateProviderImpl: usesInternalSsid is: false, usesWifi is: false
12-24 00:20:38.641 11512 22191 D ServerConnectionStateProviderImpl: usesInternalSsid is: false, usesWifi is: false
12-24 00:20:38.677 11512 22185 D ForegroundServiceLauncher: Check if service HighAccuracyLocationService is running. Service running = false
12-24 00:20:38.731 11512 11512 D LocationSensorManager: Not requesting accurate location, last accurate location was too recent
12-24 00:20:38.750 11512 10289 D ServerConnectionStateProviderImpl: usesInternalSsid is: false, usesWifi is: false
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: Error while getting core config to sync sensor status aborting
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: io.homeassistant.companion.android.common.data.integration.IntegrationException: retrofit2.HttpException: HTTP 405 Method Not Allowed
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: 	at io.homeassistant.companion.android.common.data.integration.impl.IntegrationRepositoryImpl.tryOnUrls(IntegrationRepositoryImpl.kt:691)
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: 	at io.homeassistant.companion.android.common.data.integration.impl.IntegrationRepositoryImpl.access$tryOnUrls(IntegrationRepositoryImpl.kt:70)
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: 	at io.homeassistant.companion.android.common.data.integration.impl.IntegrationRepositoryImpl$tryOnUrls$1.invokeSuspend(Unknown Source:15)
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:98)
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:124)
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: 	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:89)
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:586)
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:820)
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:717)
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:704)
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: Caused by: retrofit2.HttpException: HTTP 405 Method Not Allowed
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: 	at retrofit2.KotlinExtensions$await$2$2.onResponse(KotlinExtensions.kt:53)
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: 	at retrofit2.OkHttpCall$1.onResponse(OkHttpCall.java:164)
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: 	at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:576)
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1154)
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:652)
12-24 00:20:38.952 11512 10289 E SensorReceiverBase: 	at java.lang.Thread.run(Thread.java:1563)
12-24 00:20:38.953 11512 10289 I SensorReceiverBase: Sensor updates and sync completed
12-24 00:20:43.303 11512 11512 D LocationSensorManager$requestSingleAccurateLocation: Got single accurate location update: Location[fused 55.******,11.****** hAcc=3.79 et=+11d9h43m17s615ms alt=46.89999771118164 vAcc=1.0944078 vel=0.0 sAcc=0.24433435]
12-24 00:20:43.303 11512 11512 D LocationSensorManager$requestSingleAccurateLocation: Location accurate enough, all done with high accuracy.
12-24 00:20:43.304 11512 22185 D LocationSensorManager: Last Location: 
12-24 00:20:43.304 11512 22185 D LocationSensorManager: Coords:(55.*******, 11.*******)
12-24 00:20:43.304 11512 22185 D LocationSensorManager: Accuracy: 3.79
12-24 00:20:43.304 11512 22185 D LocationSensorManager: Bearing: 0.0
12-24 00:20:43.306 11512 22185 D LocationSensorManager: Begin evaluating if location update should be skipped
12-24 00:20:43.306 11512 22185 D LocationSensorManager: Received location that is 59 milliseconds old, 1766532043246 compared to 1766532043305 with source fused
12-24 00:20:43.309 11512 22185 D ServerConnectionStateProviderImpl: usesInternalSsid is: false, usesWifi is: false
12-24 00:20:43.432 11512 22185 E LocationSensorManager$sendLocationUpdate: Could not update location for 1.
12-24 00:20:43.432 11512 22185 E LocationSensorManager$sendLocationUpdate: io.homeassistant.companion.android.common.data.integration.IntegrationException: io.homeassistant.companion.android.common.data.integration.IntegrationException: Error calling webhook, httpCode: 405, errorBody: 405: Method Not Allowed
12-24 00:20:43.432 11512 22185 E LocationSensorManager$sendLocationUpdate: 	at io.homeassistant.companion.android.common.data.integration.impl.IntegrationRepositoryImpl.tryOnUrls(IntegrationRepositoryImpl.kt:691)
12-24 00:20:43.432 11512 22185 E LocationSensorManager$sendLocationUpdate: 	at io.homeassistant.companion.android.common.data.integration.impl.IntegrationRepositoryImpl.access$tryOnUrls(IntegrationRepositoryImpl.kt:70)
12-24 00:20:43.432 11512 22185 E LocationSensorManager$sendLocationUpdate: 	at io.homeassistant.companion.android.common.data.integration.impl.IntegrationRepositoryImpl$tryOnUrls$1.invokeSuspend(Unknown Source:15)
12-24 00:20:43.432 11512 22185 E LocationSensorManager$sendLocationUpdate: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
12-24 00:20:43.432 11512 22185 E LocationSensorManager$sendLocationUpdate: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
12-24 00:20:43.432 11512 22185 E LocationSensorManager$sendLocationUpdate: 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:124)
12-24 00:20:43.432 11512 22185 E LocationSensorManager$sendLocationUpdate: 	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:89)
12-24 00:20:43.432 11512 22185 E LocationSensorManager$sendLocationUpdate: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:586)
12-24 00:20:43.432 11512 22185 E LocationSensorManager$sendLocationUpdate: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:820)
12-24 00:20:43.432 11512 22185 E LocationSensorManager$sendLocationUpdate: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:717)
12-24 00:20:43.432 11512 22185 E LocationSensorManager$sendLocationUpdate: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:704)
12-24 00:20:43.432 11512 22185 E LocationSensorManager$sendLocationUpdate: Caused by: io.homeassistant.companion.android.common.data.integration.IntegrationException: Error calling webhook, httpCode: 405, errorBody: 405: Method Not Allowed
12-24 00:20:43.432 11512 22185 E LocationSensorManager$sendLocationUpdate: 	at io.homeassistant.companion.android.common.data.integration.impl.IntegrationRepositoryImpl$callWebhookOnUrls$response$1.invokeSuspend(IntegrationRepositoryImpl.kt:657)
12-24 00:20:43.432 11512 22185 E LocationSensorManager$sendLocationUpdate: 	... 8 more
12-24 00:21:00.512 11512 11512 D SensorReceiverBase: Received intent: android.intent.action.TIME_TICK
12-24 00:21:00.728 11512 22185 D ServerConnectionStateProviderImpl: usesInternalSsid is: false, usesWifi is: false
12-24 00:21:00.734 11512 22185 D ServerConnectionStateProviderImpl: usesInternalSsid is: false, usesWifi is: false
12-24 00:21:00.779 11512 22185 D ForegroundServiceLauncher: Check if service HighAccuracyLocationService is running. Service running = false
12-24 00:21:00.950 11512 11512 D LocationSensorManager: Received location update.
12-24 00:21:00.981 11512 22191 D ServerConnectionStateProviderImpl: usesInternalSsid is: false, usesWifi is: false
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: Error while getting core config to sync sensor status aborting
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: io.homeassistant.companion.android.common.data.integration.IntegrationException: retrofit2.HttpException: HTTP 405 Method Not Allowed
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: 	at io.homeassistant.companion.android.common.data.integration.impl.IntegrationRepositoryImpl.tryOnUrls(IntegrationRepositoryImpl.kt:691)
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: 	at io.homeassistant.companion.android.common.data.integration.impl.IntegrationRepositoryImpl.access$tryOnUrls(IntegrationRepositoryImpl.kt:70)
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: 	at io.homeassistant.companion.android.common.data.integration.impl.IntegrationRepositoryImpl$tryOnUrls$1.invokeSuspend(Unknown Source:15)
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:98)
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:124)
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: 	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:89)
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:586)
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:820)
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:717)
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:704)
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: Caused by: retrofit2.HttpException: HTTP 405 Method Not Allowed
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: 	at retrofit2.KotlinExtensions$await$2$2.onResponse(KotlinExtensions.kt:53)
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: 	at retrofit2.OkHttpCall$1.onResponse(OkHttpCall.java:164)
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: 	at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:576)
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1154)
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:652)
12-24 00:21:01.106 11512 22191 E SensorReceiverBase: 	at java.lang.Thread.run(Thread.java:1563)
12-24 00:21:01.107 11512 22191 I SensorReceiverBase: Sensor updates and sync completed

```

**Screenshot or video of problem:**

**Additional information:**

Same Issue here:

```
12-24 22:15:03.722  6830  7091 E LocationSensorManager$sendLocationUpdate: 	at io.homeassistant.companion.android.common.data.integration.impl.IntegrationRepositoryImpl.tryOnUrls(IntegrationRepositoryImpl.kt:691)
12-24 22:15:03.722  6830  7091 E LocationSensorManager$sendLocationUpdate: 	at io.homeassistant.companion.android.common.data.integration.impl.IntegrationRepositoryImpl.access$tryOnUrls(IntegrationRepositoryImpl.kt:70)
12-24 22:15:03.722  6830  7091 E LocationSensorManager$sendLocationUpdate: 	at io.homeassistant.companion.android.common.data.integration.impl.IntegrationRepositoryImpl$tryOnUrls$1.invokeSuspend(Unknown Source:15)
12-24 22:15:03.722  6830  7091 E LocationSensorManager$sendLocationUpdate: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
12-24 22:15:03.722  6830  7091 E LocationSensorManager$sendLocationUpdate: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:98)
12-24 22:15:03.722  6830  7091 E LocationSensorManager$sendLocationUpdate: 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:124)
12-24 22:15:03.722  6830  7091 E LocationSensorManager$sendLocationUpdate: 	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:89)
12-24 22:15:03.722  6830  7091 E LocationSensorManager$sendLocationUpdate: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:586)
12-24 22:15:03.722  6830  7091 E LocationSensorManager$sendLocationUpdate: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:820)
12-24 22:15:03.722  6830  7091 E LocationSensorManager$sendLocationUpdate: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:717)
12-24 22:15:03.722  6830  7091 E LocationSensorManager$sendLocationUpdate: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:704)
12-24 22:15:03.722  6830  7091 E LocationSensorManager$sendLocationUpdate: Caused by: io.homeassistant.companion.android.common.data.integration.IntegrationException: Error calling webhook, httpCode: 405, errorBody: 405: Method Not Allowed
12-24 22:15:03.722  6830  7091 E LocationSensorManager$sendLocationUpdate: 	at io.homeassistant.companion.android.common.data.integration.impl.IntegrationRepositoryImpl$callWebhookOnUrls$response$1.invokeSuspend(IntegrationRepositoryImpl.kt:657)
12-24 22:15:03.722  6830  7091 E LocationSensorManager$sendLocationUpdate: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
12-24 22:15:03.722  6830  7091 E LocationSensorManager$sendLocationUpdate: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
```

--- a/common/src/main/kotlin/io/homeassistant/companion/android/common/data/servers/ServerConnectionStateProviderImpl.kt
+++ b/common/src/main/kotlin/io/homeassistant/companion/android/common/data/servers/ServerConnectionStateProviderImpl.kt
@@ -109,8 +109,8 @@ class ServerConnectionStateProviderImpl @AssistedInject constructor(
                 connection.internalHttpUrl?.buildWebhookUrl(webhookId)?.let(::add)
             }
 
-            // Cloud URL: always add if available
-            connection.cloudHttpUrl?.let(::add)
+            // Cloudhook URL: always add if available
+            connection.cloudhookHttpUrl?.let(::add)
 
             // External URL: use when on home network, or when connection is secure/allowed
             val externalUrlIsSecure = connection.externalUrl.startsWith("https://")

--- a/common/src/test/kotlin/io/homeassistant/companion/android/common/data/servers/ServerConnectionStateProviderImplTest.kt
+++ b/common/src/test/kotlin/io/homeassistant/companion/android/common/data/servers/ServerConnectionStateProviderImplTest.kt
@@ -534,27 +534,29 @@ class ServerConnectionStateProviderImplTest {
         }
 
         @Test
-        fun `Given not on home network and has cloudUrl when calling getApiUrls then cloudUrl is first`() = runTest {
+        fun `Given not on home network and has cloudhookUrl when calling getApiUrls then cloudhookUrl is first`() = runTest {
             val provider = createServerConnectionStateProvider(
                 externalUrl = "https://external.example.com",
                 internalUrl = "http://192.168.1.1:8123",
-                cloudUrl = "https://cloud.nabu.casa/abc123",
+                cloudUrl = "https://abc123.ui.nabu.casa",
+                cloudhookUrl = "https://hooks.nabu.casa/cloudhook123",
                 webhookId = "webhook123",
             )
 
             val result = provider.getApiUrls()
 
             assertEquals(2, result.size)
-            assertEquals("https://cloud.nabu.casa/abc123", result[0].toString())
+            assertEquals("https://hooks.nabu.casa/cloudhook123", result[0].toString())
             assertEquals("https://external.example.com/api/webhook/webhook123", result[1].toString())
         }
 
         @Test
-        fun `Given on home network with cloudUrl when calling getApiUrls then internal is first followed by cloudUrl`() = runTest {
+        fun `Given on home network with cloudhookUrl when calling getApiUrls then internal is first followed by cloudhookUrl`() = runTest {
             val provider = createServerConnectionStateProvider(
                 externalUrl = "https://external.example.com",
                 internalUrl = "http://192.168.1.1:8123",
-                cloudUrl = "https://cloud.nabu.casa/abc123",
+                cloudUrl = "https://abc123.ui.nabu.casa",
+                cloudhookUrl = "https://hooks.nabu.casa/cloudhook123",
                 webhookId = "webhook123",
                 internalEthernet = true,
             )
@@ -564,7 +566,7 @@ class ServerConnectionStateProviderImplTest {
 
             assertEquals(3, result.size)
             assertEquals("http://192.168.1.1:8123/api/webhook/webhook123", result[0].toString())
-            assertEquals("https://cloud.nabu.casa/abc123", result[1].toString())
+            assertEquals("https://hooks.nabu.casa/cloudhook123", result[1].toString())
             assertEquals("https://external.example.com/api/webhook/webhook123", result[2].toString())
         }
 

